app/serializers/ticket_dispenser/ticket_serializer.rb

```diff
@@ -6,7 +6,10 @@ class TicketSerializer < ActiveModel::Serializer
 
     def sender
       return if object.messages.empty?
-      object.messages.first.sender&.username
+      {
+        username: object.messages.first.sender&.username,
+        real_name: object.messages.first.sender&.real_name
```

Comment:
This couples it more tightly to the specific shape of our `User` model, although it was already couple with `username` before. Now's the time to choose whether we want this to potentially be useful for other apps, or whether it's just a way to pull out features. If the former, we should probably rely on the main app to provide user serialization.